src/test/resources/log4j2-test.properties

```diff
@@ -0,0 +1,19 @@
+status = error
```

Comment:
How is this file used?

Reply:
Initially I thought that RMB library leveraging this config somehow without specifying it as log4j.configurationFile property. But later as in stated in the https://issues.folio.org/browse/RMB-801 this config unused finally. So this can be skipped.

Reply:
Thank you for investigating this.